app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala

```diff
@@ -54,6 +54,10 @@ class FrontendAppConfig @Inject() (servicesConfig: ServicesConfig) {
 
   val reminderServiceFeatureSwitch: Boolean = servicesConfig.getBoolean("reminder-feature-switch")
 
+  val moneyHelper: String = servicesConfig.getString("money-helper.url")
+
+  val manageDetails: String = servicesConfig.getString("mange-details.url")
```

Comment:
Is "mange" spelt correctly? Should it be "manage" instead

Reply:
Apologies, that is a misspelling on my part.